internal/gengapic/gengapic_test.go

```diff
@@ -499,3 +500,86 @@ func Test_isLRO(t *testing.T) {
 		}
 	}
 }
+
+func Test_transportParse(t *testing.T) {
```

Comment:
maybe change the name to `Test_optionParse` since you're testing other options as well.

Reply:
Yeah, the scope of the test really expanded.